### ISSUE TITLE
Numbered list ignores weighting

### ISSUE THREAD

**Issue by mxdvl** [guardian/dotcom-rendering#5353]

**Comment by bryophyta** [comment]
Update: another email came through related to this issue yesterday. It sounds like there's a relatively easy fix in the editorial tools, so perhaps if we aren't able to fix this in the near future we could try to publicise the tools-side fix?

**Comment by paperboyo** [comment]
> we could try to publicise the tools-side fix

Not really a fix as much as users having to remember to change tools switches that are ignored by DCR. They will never remember to do it, even more so because, apart from an image going blurry, the preview looks exactly the same.

What happens is, Design decided that Numbered list template is rendered with always effectively a Showcase Main media weighting no matter what is the state of weighting switch in Composer. So a platform is making a decision of ignoring a switch. Always dodgy idea (think what happens in ten years with those*), but if upheld – the platform in question should make sure it’s requesting a correctly dimensioned image (effectively a Showcase-sized one).

*platforms ignoring switches inevitably leads to inconsistencies in legacy content which will become visible once the design changes. Some of the Numbered list articles will have this “fixed” (Showcase), some not (“Inline”). One day, when platform(s) will go back to honouring content switches, they will look… random.

/end rant, super-minor, but wanted to make the point

**Comment by HarryFischer** [comment]
Is a solution to this to stop forcing showcase main media? Allow inline and showcase?

**Comment by oliverlloyd** [comment]
> Is a solution to this to stop forcing showcase main media? Allow inline and showcase?

For me, this makes sense. In composer, `numberedList` is a display hint which mainly affects how ul/li tags are rendered. But it also has the effect of forcing the main media to showcase, but this feels wrong. Ideally, we'd respect the weighting choice made in Composer and allow the main media to be inline as well as showcase.

If we did this, then there's a case to be made that `Display.NumberedList` should be migrated to `Design.Numbered` list. This would be a lot more semantic - it never felt right being able to combine the `NumberedList` display type with other designs. Whereas having a showcase numbered list or an immersive numbered list both feel like good ways to extend the model.

**Comment by paperboyo** [comment]
> For me, this makes sense

There was this thinking of moving Main media weighting to live where Display hints live. This would prevent it being possible. Just saying. cc @HarryFischer

**Comment by paperboyo** [comment]
I agree it could (and should) be given some thought (rethought). I agree that developing `format` helped unearth some stupidity. But I don’t actually agree, now that you have developed `format`, Tools should blindly follow. For example, inventing PrintShop was just a (cheeky) way of you not being able to (deal with a) design for first image in the body set to Immersive. Tools cannot care about that, nor should, I don’t think. First image just can be Immersive, tough. 😜 

`format` mixes some other concepts, too, IIRC.

> just give _all_ articles the same features that photo essays have?

💯

**Comment by oliverlloyd** [comment]
For sure there are some aspects of `format` that are not ideal, `PrintShop` is one, `NumberedList` is another. But equally, the tools also have issues and inconsistencies, so we're both as bad as each other.

I don't think either side should be following or directing the other. Instead, we should be working _together_ to align and solve these problems to make things better. I realise this sounds like a speech from an 1990s inspirational movie but it really does feel like there's an opportunity here.

**Comment by paperboyo** [comment]
💯 
BTW, I loved that movie!

![image](https://user-images.githubusercontent.com/6032869/187460028-1c37731e-0909-4a53-800a-c1375c686281.png)
